### DISCUSSION TITLE
vue2的项目将部分组件通过cdn方式引入，还需要在package.json的 dependencies 中引用吗？

### DISCUSSION THREAD

**Discussion by yesay** [empjs/emp#309] [Q&A] [answered]
vue2的项目将部分组件通过cdn方式引入，还需要在package.json的 dependencies 中引用吗？

如果不在 package.json的 dependencies 中引用，打包 emp build 时会报错。
![image](https://user-images.githubusercontent.com/3607509/187846815-5f701916-19e5-4ddc-bc63-d4616194b545.png)
![image](https://user-images.githubusercontent.com/3607509/187846928-8ee811d9-6781-4c67-8c14-7c9b805c249d.png)

**Comment by yesay** [comment]
通过cdn方式引入的意义是什么？

**Comment by ckken** [comment, marked as answer]
参考 三级共享 https://juejin.cn/post/7051492163934224397
